### PR TITLE
Little King's Story PCKG support

### PR DESCRIPTION
**Pull request by iwubcode**

### CODE REVIEW

**Comment by iwubcode** [comment]
Get's LKS to 4228 or 96%

File: lib/AuroraLip/Archives/Formats/PCKG_CING.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+namespace AuroraLip.Archives.Formats
+{
+    class PCKG_CING : Archive, IFileAccess
+    {
+        public bool CanRead => true;
+
+    public bool CanWrite => false;
+
+    private const string Extension = "PCKG";
```
Comment (Venomalia) [inline lib/AuroraLip/Archives/Formats/PCKG_CING.cs:13]: this is the magic? why don't you use IMagicIdentify.

Comment (Venomalia) [reply lib/AuroraLip/Archives/Formats/PCKG_CING.cs]: should look like this
```
    public class PCKG : Archive, IMagicIdentify, IFileAccess
    {
    
        public string Magic => magic;

        private const string magic = "PCKG";
```

File: lib/AuroraLip/Archives/Formats/PCKG_CING.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
+﻿using AuroraLip.Common;
+using System;
+using System.IO;
+
+namespace AuroraLip.Archives.Formats
+{
+    class PCKG_CING : Archive, IMagicIdentify
```
Comment (Venomalia) [inline lib/AuroraLip/Archives/Formats/PCKG_CING.cs:7]: public class PCKG_CING : Archive, IMagicIdentify, IFileAccess

File: lib/AuroraLip/Archives/Formats/PCKG_CING.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        public string Magic => magic;
+
+        private const string magic = "PCKG";
+
+        public PCKG_CING() { }
+
+        public PCKG_CING(string filename) : base(filename) { }
+
+        public PCKG_CING(Stream stream, string filename = null) : base(stream, filename) { }
+
```
Comment (Venomalia) [inline lib/AuroraLip/Archives/Formats/PCKG_CING.cs:22]: public bool IsMatch(Stream stream, in string extension = "")
            => stream.MatchString(magic);

File: lib/AuroraLip/Common/FormatDictionary_List.cs [hunk 4]
```diff
@@ -233,6 +233,9 @@ public static partial class FormatDictionary
             //H.a.n.d.
             new FormatInfo(".fbc", FormatType.Archive, "Fables Chocobo archive", "H.a.n.d.") { Class= typeof(FBC)},
 
+            //Cing
+            new FormatInfo(".pac", "PCKG", FormatType.Archive, "Little King's Story Archive", "Cing"){ Class= typeof(PCKG_CING)},  // also pcha
```
Comment (Venomalia) [inline lib/AuroraLip/Common/FormatDictionary_List.cs:237]: `new FormatInfo(".pac", "PCKG", FormatType.Archive, "Little King's Story Archive", "Cing"),`

if we use IMagicIdentify and IFileAccess the program will assign the class automatically.

**Review by Venomalia — COMMENTED** [review]
I would still like to ask you to adjust this.

would it be ok if we just call the class PCKG?

**Pull request merged**